src/Development/IDE/GHC/CPP.hs

```diff
@@ -139,7 +141,11 @@ getBackendDefs :: DynFlags -> IO [String]
 getBackendDefs dflags | hscTarget dflags == HscLlvm = do
     llvmVer <- figureLlvmVersion dflags
     return $ case llvmVer of
-#if MIN_GHC_API_VERSION(8,8,0)
+#if MIN_GHC_API_VERSION(8,8,2)
+               Just v
+                 | m:n:_ <- llvmVersionList v -> [ "-D__GLASGOW_HASKELL_LLVM__=" ++ format (m, n) ]
+                 | n:_   <- llvmVersionList v -> [ "-D__GLASGOW_HASKELL_LLVM__=" ++ format (n, 0) ]
```

Comment:
I don’t have any projects which use LLVM, so I’d appreciate it if someone could double-check that this is doing what they expect.

Note that despite `llvmVersionList` returning `[Int]`, these branches are in fact exhaustive (under `ghc` 8.8.2), because `LlvmVersion` contains a `NonEmpty` list and converts it to a list otherwise unmodified. So we will always have at least a major version, and thus this will not silently fail (modulo future changes to `ghc`’s API).